src/main/java/org/opentripplanner/apis/support/TileJson.java

```diff
@@ -96,4 +93,14 @@ public static String urlFromOverriddenBasePath(
         String.join(",", layers)
       );
   }
+
+  private static String attributionFromFeedInfo(Collection<FeedInfo> feedInfos) {
+    return feedInfos
+      .stream()
+      .map(feedInfo ->
+        "<a href='" + feedInfo.getPublisherUrl() + "'>" + feedInfo.getPublisherName() + "</a>"
```

Comment:
Could we use `String.format`?

Reply:
e55c2c91d234de98f15dde60401823bf051f23ff